string directory
string save_method
int32 recording_length # [s]
---
bool success
